prime-router/src/main/kotlin/azure/ActionHistory.kt

```diff
@@ -203,6 +213,14 @@ class ActionHistory {
             action.actionResponse = JSONB.valueOf(verboseResponse)
     }
 
+    /**
+     * Set the action response. This is primarily used by backend functions that do not have an associated HttpRequest
+     * @param jsonResponse the generated stringified json representing the response
+     */
+    fun trackActionResponse(jsonResponse: String) {
+        action.actionResponse = JSONB.valueOf(jsonResponse)
```

Comment:
This code is a duplicate of line 213.  Can you go back and call this function where this code is used?

Reply:
This is an overload of the function call on 213 - we don't have a HttpResponseMessage in this case.

Reply:
Good call. Updated.